### PR TITLE
(chore): Removed registration for new body from outside

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
We have agreed on registration for bodies from outside we use the normal patient registration

## Screenshots
Peek 2024-10-09 21-31

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-morgue-app/src/index.ts [hunk 1]
```diff
@@ -33,4 +33,7 @@ export const morgueDashboardLink = getSyncLifecycle(
   options,
 );
 
-export const morgueEnrollWorkspace = getAsyncLifecycle(() => import('./workspaces/enrollBody.workspace'), options);
+export const addMoreDetailsWorkspace = getAsyncLifecycle(
```
Comment (donaldkibet) [inline packages/esm-morgue-app/src/index.ts:36]: should we rename this form with a more descriptive name. more details is too generic e.g `patientAdditionalInfoForm`

File: packages/esm-morgue-app/src/routes.json [hunk 2]
```diff
@@ -23,9 +23,9 @@
   ],
    "workspaces": [
     {
-      "name": "morgue-enroll-form",
-      "component": "morgueEnrollWorkspace",
-      "title": "Enroll new deceased",
+      "name": "add-more-details-form",
```
Comment (donaldkibet) [inline packages/esm-morgue-app/src/routes.json:26]: same as here

File: packages/esm-morgue-app/src/routes.json [hunk 3]
```diff
@@ -23,9 +23,9 @@
...
+      "component": "addMoreDetailsWorkspace",
+      "title": "add more details",
```
Comment (donaldkibet) [inline packages/esm-morgue-app/src/routes.json:28]: we should also try translate the hard coded values here e using `translateFrom` function

File: packages/esm-morgue-app/src/tables/waiting-queue.component.tsx [hunk 4]
```diff
@@ -54,10 +56,14 @@ export const WaitingQueue: React.FC = () => {
       Ids: patient.identifiers[0]?.identifier || 'N/A',
       address4: patient.person.preferredAddress?.address4 || 'N/A',
     })) || [];
-
+  const handleMoreDetails = () => {
```
Comment (donaldkibet) [inline packages/esm-morgue-app/src/tables/waiting-queue.component.tsx:59]: should this be `admission-workspace` instead of `add-more-details-form`

**Review by donaldkibet — CHANGES_REQUESTED** [review]

File: packages/esm-morgue-app/src/index.ts (hunk 1 above)
Comment (its-kios09) [reply packages/esm-morgue-app/src/index.ts]: sorted

**Comment by its-kios09** [comment]
@donaldkibet  Thanks i have sorted it out

**Review by donaldkibet — APPROVED** [review]

**Pull request merged**